HSTracker/Utility/BoardDamage/BoardHero.swift

```diff
@@ -12,24 +12,47 @@ class BoardHero: IBoardEntity {
     private(set) var _baseAttack = 0
     private(set) var _hero: BoardCard
     private(set) var _weapon: BoardCard?
-    
-    var name: String { return _hero.name }
-    var hasWeapon: Bool { return _weapon != nil }
-    
+
+    var name: String {
+        return _hero.name
+    }
+    var hasWeapon: Bool {
+        return _weapon != nil
+    }
+
+    var heroPowerAttack: Int {
+        switch _hero._class {
+        case "MAGE":
+            return 1
+        case "HUNTER":
+            return 2
+        default:
+            return 0
```

Comment:
Mmmm true, it would be better to get the hero power token instead, I was just figuring things out don't mind this part